platform/MCU/STM32F2xx/SPARK_Firmware_Driver/src/system_flags_impl.c

```diff
@@ -0,0 +1,79 @@
+#include "system_flags_impl.h"
+
+#include "stm32f2xx_rtc.h"
+
+#include <string.h>
+
+#define SYSTEM_FLAGS_MAGIC_NUMBER 0x1ADEACC0u
+
+int Load_SystemFlags_Impl(platform_system_flags_t* f) {
```

Comment:
Fixed in `Load_SystemFlags()` and `Save_SystemFlags()` which call these `_Impl` functions.